slack/main.go

```diff
@@ -330,3 +331,27 @@ buildLoop:
 	attachmentMsgOption := slack.MsgOptionAttachments(attachment)
 	return &attachmentMsgOption
 }
+
+// getMostRecentBuilds finds the most recent builds for each build trigger
+// triggers being a build or deploy trigger or some kind of build event trigger
+func getMostRecentBuilds(builds map[string]*cbpb.Build) map[string]*cbpb.Build {
```

Comment:
**Thought (Non Blocking):**
Bummer we have to just loop through all these and aren't able to provide some sort of filter when retrieving stored builds in the first place, but I took a quick glance at the SDK and doesn't look like it's possible to do that kinda filtering, so a well-commented loop looks good to me!

Reply:
Awesome, thanks for looking too! 🙇